scripts/ci-test.sh

```diff
@@ -22,4 +22,11 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 # shellcheck source=../hack/ensure-go.sh
 source "${REPO_ROOT}/hack/ensure-go.sh"
 
-cd "${REPO_ROOT}" && make generate lint verify test
+cd "${REPO_ROOT}" && make generate template lint verify test
```

Comment:
```suggestion
cd "${REPO_ROOT}" && make generate templates lint verify test
```

Will fix the regeneration

Reply:
Thanks, fixed!